Exercises/VincentWieg/exercises_05.py

```diff
@@ -0,0 +1,126 @@
+# PART 1:
+# Here's a sequence of numbers:
+# [35, 20, 15, 25, 47, 40, 62, 55, 65, 95, 102, 117, 150, 182, 127, 219, 299, 277, 309, 576]
+# numbers in this list can in general be expressed as a sum of some pair of two numbers
+# in the five numbers preceding them.
+# For example, the sixth number (40) cam be expressed as 25 + 15
+# the seventh number (62) can be expressed as 47 + 15 etc.
+# 
+# The only exception to this rule for this example is the number 127.
+# The five preceding numbers are [95, 102, 117, 150, 182], and no possible sum of two of those
+# numbers adds to 127.
+#
+# You can find the ACTUAL input for this exercise under `data/input_sequence.txt`. For this
+# real input you should consider not only the 5 numbers, but the 25 numbers preceding.
+# Find the first number in this list which can not be expressed as a
+# sum of two numbers out of the 25 numbers before it.
+# Please make not of your result in the PR.
+import os
+
+
+
+with open("data\input_sequence.txt", "r") as file:
+    file_content = file.read()
+
+file_content_list = list(map(int, file_content.split()))
+
+
+#inputSequence = [35, 20, 15, 25, 47, 40, 62, 55, 65, 95, 102, 117, 150, 182, 127, 219, 299, 277, 309, 576]
+
+def sequenceChecker(inputSequence):
+    numbers_can_be_added = 25 
+
+    for index_of_digit_to_check in range(numbers_can_be_added, len(inputSequence)):
+        digit_to_check = inputSequence[index_of_digit_to_check]
+        list_to_check = inputSequence[index_of_digit_to_check - numbers_can_be_added : index_of_digit_to_check]
+
+        found_sum = False
+        for i in range(len(list_to_check)):
+            for j in range(i + 1, len(list_to_check)): 
+                if list_to_check[i] + list_to_check[j] == digit_to_check:
+                    found_sum = True
+                    break
+            if found_sum:
+                break
+
+
+        if found_sum == False:
```

Comment:
Anstatt if found_sum == False: wäre es eleganter die abfrage wie folgt zu verändert:
```suggestion
        if not found_sum:
```
Dies erfüllt den selben Zweck.

Reply:
PEP8: if you keep using your old variable use is instead of ==